Xamarin.PropertyEditing.Mac/PropertyTableDelegate.cs

```diff
@@ -184,6 +184,13 @@ public override void ItemDidCollapse (NSNotification notification)
 			}
 		}
 
+		public override void DidRemoveRowView (NSOutlineView outlineView, NSTableRowView rowView, nint row)
+		{
+			if (rowView.Subviews[0] is EditorContainer ec) {
+				ec.ViewModel = null;
+			}
+		}
```

Comment:
As we are overriding, don't we want to call `base.DidRemoveRowView()` too?

Reply:
There is no base for native types like this, if you call it it will throw.